PWGEM/PhotonMeson/TableProducer/skimmerPrimaryElectron.cxx

```diff
@@ -68,8 +68,7 @@ struct skimmerPrimaryElectron {
   Configurable<float> maxeta{"maxeta", 0.9, "eta acceptance"};
   Configurable<float> dca_xy_max{"dca_xy_max", 1.0f, "max DCAxy in cm"};
   Configurable<float> dca_z_max{"dca_z_max", 1.0f, "max DCAz in cm"};
-  Configurable<float> min_tpcdEdx{"min_tpcdEdx", 40.0, "min TPC dE/dx"};
-  Configurable<float> max_tpcdEdx{"max_tpcdEdx", 110.0, "max TPC dE/dx"};
+  Configurable<float> minTPCNsigmaEl{"minTPCNsigmaEl", -4.0, "max. TPC n sigma for electron inclusion"};
```

Comment:
comment here should also be `min` instead of `max`

Reply:
thank you. I will fix it in the next iteration.